src/Sylius/Component/Core/Repository/OrderRepositoryInterface.php

```diff
@@ -19,7 +20,18 @@
 
 interface OrderRepositoryInterface extends BaseOrderRepositoryInterface
 {
+    /**
+     * @return QueryBuilder
+     */
     public function createListQueryBuilder();
+    
+    /**
+     * @param CustomerInterface $customer
+     * @param array $sorting
+     *
+     * @return QueryBuilder
+     */
+    public function createQueryBuilderWithCustomer(CustomerInterface $customer, array $sorting = []);
```

Comment:
@Zales0123 To me both ``createByCustomerQueryBuilder`` and ``createQueryBuilderByCustomer`` sound good, but I am leaning towards the former.